src/testcases/org/apache/poi/xssf/usermodel/helpers/XSSFColumnShifterTest.java

```diff
@@ -0,0 +1,384 @@
+package org.apache.poi.xssf.usermodel.helpers;
+
+import static org.apache.poi.POITestCase.skipTest;
+import static org.apache.poi.POITestCase.testPassesNow;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.junit.Assume.assumeTrue;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.poi.common.usermodel.HyperlinkType;
+import org.apache.poi.ss.ITestDataProvider;
+import org.apache.poi.ss.usermodel.*;
+import org.apache.poi.ss.util.CellAddress;
+import org.apache.poi.ss.util.CellRangeAddress;
+import org.apache.poi.ss.util.CellUtil;
+import org.apache.poi.xssf.XSSFITestDataProvider;
+import org.apache.poi.xssf.XSSFTestDataSamples;
+import org.apache.poi.xssf.usermodel.*;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class XSSFColumnShifterTest {
+
+    //private static Logger log = LoggerFactory.getLogger(XSSFColumnShifterTest.class + "_T");
+    private XSSFSheet sheet1, sheet2;
+    private Workbook wb07;
+
+    protected final ITestDataProvider _testDataProvider;
+
+    public XSSFColumnShifterTest(){
+        _testDataProvider = XSSFITestDataProvider.instance; 
+    }
+
+    @Before
+    public void init() {
+        wb07 = new XSSFWorkbook();
```

Comment:
If you want, you can load a test spreadsheet from test-cases/spreadsheet, saving you the setup code here.